Windows support for Passwd plugin (#1516)
chef/ohai: RELEASE_NOTES.md

# Unreleased

## etc Ohai Data on Windows

> Ohai's 'passwd' plugin that provides `node['etc']['passwd']` and `node['etc']['group']` now populates data on Windows. Data for all local users and groups is present. A few things to note:

Reviewer: ```suggestion
Ohai's 'Passwd' plugin that provides `node['etc']['passwd']` and `node['etc']['group']` now populates data on Windows. Data for all local users and groups is present. A few things to note:
```

>   * If you are on a domain controller, you will get all domain users as domain controllers see domain users as local

Reviewer: ```suggestion
  * If the node is an Active Directory domain controller, you will get all domain users as domain controllers see domain users as local
```

>   * If you are not on a domain controller you will only get actual local users

Reviewer: ```suggestion
  * If the node is not an Active Directory domain controller you will only get actual local users
```
